Intro to Python Notebook #1 .ipynb

```diff
@@ -0,0 +1,1718 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Intro to Python Notebook #1 \n",
+    "\n",
+    "The first notebook in a series of tutorial intro to python notebooks. \n",
+    "\n",
+    "author: Emily PL, elp25@duke.edu \n",
+    "last tested: \n",
+    "\n",
+    "Topics covered: \n",
+    "\n",
+    "* installing python and jupyter \n",
+    "* running python\n",
+    "* variables\n",
+    "\n",
+    "TODO: add exercises\n",
+    "\n",
+    "\n",
+    "References: *Python Data Science Handbook*, Jake VanderPlas. *Effective Computation in Physics*, Anthony Scopatz & Katheryn D. Huff "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part 0: Installing Python and Jupyter \n",
+    "\n",
+    "This notebook can be run on your laptop, but you will need python and Jupyter. \n",
+    "The recommended way to setup python is with Anaconda.  Installation instructions for different operating systems can be found here: https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html \n",
+    "\n",
+    "You can then launch Jupyter by typing:\n",
+    "<br>\n",
+    "jupyter notebook \n",
```

Comment:
Being picky, but I would say it is better to mark in the text whenever we are using a command from the terminal or the interpreter with markdown using ``.  For example in this case, we would have `jupyter notebook` instead of jupyter notebook.
Also, in case of a terminal unix command I would include the cursor (tipically the  $ or >).